src/formatDate.js

```diff
@@ -50,7 +50,52 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const parts = date.split(fromFormat[fromFormat.length - 1]);
+  let newDate = '';
+
+  const YEAR_FULL = 'YYYY';
+  const YEAR_SHORT = 'YY';
+  const MONTH = 'MM';
+  const DAY = 'DD';
+
+  for (let i = 0; i < toFormat.length; i++) {
+    switch (true) {
+      case toFormat[i].includes(YEAR_SHORT):
+        if (fromFormat.includes(YEAR_FULL) && toFormat[i] === YEAR_SHORT) {
```

Comment:
The 'includes' function is used multiple times for the same variables, which is not efficient. Consider assigning the result to a variable and reusing it instead.